FIRO_TSEnsembles/src/main/java/hec/stats/MaxComputable.java

```diff
@@ -0,0 +1,20 @@
+package hec.stats;
+
+public class MaxComputable implements Computable{
+    @Override
+    public float compute(float[] values){
+        //calculate the min of values
```

Comment:
Please Remove Commented Code

Reply:
but what if I think I want to use it later?